src/v2/components/screens/Start/styles.ts

```diff
@@ -0,0 +1,227 @@
+import styled from "styled-components";
+import { EasyMintLogo as _EasyMintLogo } from "../../EasyMintLogo";
+import { Whale as _Whale } from "../../Whale";
+import { ButtonLink } from "../../Button";
+
+export const Button = styled(ButtonLink)`
+  margin-top: 48px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 24px 48px;
+  gap: 12px;
+
+  width: 217px;
+  height: 60px;
+
+  background: #f2f6f7;
+  border: 1px solid #fcfdfd;
+  box-shadow: 0px 8px 0px -4px #030303;
+  border-radius: 4px;
+
+  font-family: "New York";
+  font-style: normal;
+  font-weight: 810;
+  font-size: 20px;
+  line-height: 12px;
+  /* identical to box height, or 60% */
+
+  text-align: center;
+  letter-spacing: 0.0125em;
+
+  color: #0a0a0b;
+`;
+
+export const Center = styled.div`
+  margin-top: 200px;
+  align-items: flex-start;
+  display: flex;
+  flex-direction: column;
+  max-width: 620px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+`;
+
+export const Container = styled.article`
+  height: 100%;
+  width: 100%;
+  background-image: url("/landing_background.svg");
```

Comment:
is SVG the best file format for this? i'm not sure, just asking as the image seems quite complex

Reply:
It is actually an `svg` with png embedded for each different avatar.
I complete png may be better here but I could not export that from figma designs. I will check with Max if he can provide that.

Reply:
@mjmayank : I updated this image in the following PR: https://github.com/Wally-xyz/twitter-demo-frontend/pull/65